Can't include the library in node.js after installing it with npm

The following line, in node.js, after successfully npm installing the package `@alexaltea/capstone-js` :
```js
const cs = require("@alexaltea/capstone-js")
```

results in an error :
```
Error: Cannot find module '@alexaltea/capstone-js'
```

--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
     "type": "git",
     "url": "https://github.com/AlexAltea/capstone.js.git"
   },
+  "main": "./dist/capstone.min.js",
   "dependencies": {},
   "devDependencies": {
     "bower": "^1.8.8",

